document/issuance.md

```diff
@@ -0,0 +1,70 @@
+# About Issuance
+
+## What is Issuance
+
+Issuance is used to issue digital credentials, structured according to the selected credential schema, and deliver them to recipients via email. When performed as a batch based on a CSV file, it saves a tremendous amount of time and operationalises the task of issuing and delivering digital credentials.
+
+## How does Issuance work
+
+Let us take a look under the hood. Firstly, an issuance “container” is created, which is defined by the credential schema selected, your project’s issuer DID used, a wallet URL and a timestamp. Secondly, individual credential offers are added to it. As these credential offers do not contain holder DIDs at this point, they are not yet the digital credentials. 
+
+In order to finalise the credential issuance process, emails are sent out to the recipients. They receive a link in their mailbox to a wallet with a Credential Offer Request Token. When clicking this link they will be redirected to a wallet that generates a Credential Offer Response Token and uses it to claim the digital credential. A successful login to the wallet implicitly proves that a recipient has access to the specified email and provides the missing holder DID information necessary to issue the digital credential.
+
+## Frequently asked questions
+
+### Where can I find a schema URL?
+
+Issuance works best with the schemas stored under VC Schemas. You can use either JSON schema URL or JSON-LD context URL of the selected schema. Here are examples of both URLs for a PartyInvite credential: 
+
+- JSON schema URL: `https://schema.affinidi.com/PartyInviteV1-0.json`
+- JSON-LD context URL: `https://schema.affinidi.com/PartyInviteV1-0.jsonld`
```

Comment:
I usually use `MySchemaV1-0`, cause it's simple and up-to-date (this schema seems to be outdated)

We also use MySchema in extension for examples